apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -61,6 +61,7 @@ const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfoWithText} = {
     },
   },
   [BlockTypes.SET_VOLUME_EFFECT]: {
+    levelbuilderText: 'Set volume effect',
```

Comment:
Thanks for adding this! Guessing it would disappear from this diff if the branch was updated? https://github.com/code-dot-org/code-dot-org/pull/61100

Reply:
Yup, resolved by merging latest `staging`.